conans/client/deps_builder.py

```diff
@@ -213,12 +214,13 @@ def private_nodes(self):
 class DepsBuilder(object):
     """ Responsible for computing the dependencies graph DepsGraph
     """
-    def __init__(self, retriever, output):
+    def __init__(self, retriever, output, loader):
```

Comment:
DepsBuilder was using Loader indirectly through "Retriever" instance. Now, retriever no longer need a Loader, so make it explicit for DepsBuilder